icons/adobephotoshop.svg

```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-adobephotoshop-icon" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title id="simpleicons-adobephotoshop-icon">jQuery icon</title><path d="M.5.8v22.4h23V.8H.5zm1 .9h21.1v20.5H1.5V1.7zM6.1 6c.1-.1.2-.1.2-.1H8.7c1.3 0 2.3.3 2.8 1s.9 1.4.9 2.4c0 1.2-.4 2.1-1.1 2.7s-1.7.8-2.8.8h-.7V16.6H6.2h-.1V6zm1.6 5.1h.7c.6 0 1.1-.1 1.5-.4.4-.3.7-.8.7-1.5 0-.6-.2-1.1-.5-1.4-.3-.2-.8-.4-1.4-.4h-.9v3.7h-.1zM16 10c-.4 0-.6.1-.8.2-.1.2-.2.3-.2.5s.1.3.2.5.5.4 1 .6c.8.4 1.3.7 1.6 1.1.3.4.4.8.4 1.3 0 .8-.3 1.3-.8 1.7s-1.2.6-2 .6c-.4 0-.8 0-1.1-.1-.3-.1-.6-.2-.8-.3 0 0-.1 0-.1-.1v-1.5-.1h.1c.3.2.6.4.9.4s.7.1 1 .1c.4 0 .7-.1.8-.2.2-.1.3-.3.3-.5s-.1-.3-.2-.5c-.2-.2-.5-.4-1-.6-.7-.3-1.2-.7-1.5-1.1-.3-.4-.4-.8-.4-1.4 0-.6.2-1.1.7-1.6s1.1-.7 1.9-.7c.4 0 .8 0 1.1.1.3.1.5.1.7.2l.1.1V10.2h-.1c-.2-.1-.5-.2-.8-.3-.4.1-.7.1-1 .1z"/></svg>
```

Comment:
`<title>jQuery icon</title>` should be changed to Adobe Photoshop

Reply:
I'm an idiot